nanostack-event-loop/platform/arm_hal_timer.h

```diff
@@ -16,7 +16,7 @@
 #ifndef ARM_HAL_TIMER_H_
 #define ARM_HAL_TIMER_H_
 
-#ifdef MBED_CONF_NANOSTACK_EVENTLOOP_USE_PLATFORM_TICK_TIMER
+#if MBED_CONF_NANOSTACK_EVENTLOOP_USE_PLATFORM_TICK_TIMER
 #define NS_EVENTLOOP_USE_TICK_TIMER MBED_CONF_NANOSTACK_EVENTLOOP_USE_PLATFORM_TICK_TIMER
```

Comment:
Normally internally we use ifdef, so I had in mind

#if MBED_CONF_NANOSTACK_EVENTLOOP_USE_PLATFORM_TICK_TIMER
#define NS_EVENTLOOP_USE_TICK_TIMER 1
#endif

And then use ifdef with NS_EVENTLOOP_USE_TICKER_TIMER